docs/cli/gittuf_clone.md

```diff
@@ -9,8 +9,9 @@ gittuf clone [flags]
 ### Options
 
 ```
-  -b, --branch string   specify branch to check out
-  -h, --help            help for clone
+  -b, --branch string      specify branch to check out
+  -h, --help               help for clone
+  -k, --root-key strings   path(s) of root keys to verify when cloning the repo.
```

Comment:
```suggestion
  -b, --branch string    specify branch to check out
  -h, --help             help for clone
  -k, --root-key paths   root keys to verify when cloning the repo.
```

Reply:
@neilnaveen can you check if cobra allows us to annotate the type for this?

@flandweber this is auto generated by cobra, so it uses the underlying datatype.

Reply:
I have fixed this, I did not use cobra, I just created a custom type, which made cobra display it as "paths"